src/mgr/MgrOpRequest.h

```diff
@@ -52,7 +52,6 @@ struct MgrOpRequest : public TrackedOp {
 
 public:
   ~MgrOpRequest() override {
-    request->put();
```

Comment:
Oh, it looks the intention was rather `request.reset()` or, as this patch proposes, to delegate the cleaning to the destructor of the `request` itself.

nit: can we have `~MgrOpRequest` defaulted (`= default`)?

Reply:
Thanks for the review. Yes. We can do that.